README.md

```diff
@@ -6,7 +6,21 @@ This project contains the `moactl` command line tool that simplifies the use of
 
 This guide walks through setting up your first Amazon Red Hat OpenShift cluster using `moactl`.
 
-If you have already [installed the required prerequisites](#Installation-prerequisites), here are the commands you need to create a cluster.
+Before you can start working with `moactl` commands you have to have your AWS privileges set up correctly.
+To do this, run the following command:
+
+```
+$ aws iam attach-user-policy --user-name <username> --policy-arn "arn:aws:iam::aws:policy/AdministratorAccess"
```

Comment:
This is the same as #52 I think we should inform the user that they should add the policy because its likely they can't add it themself.

Reply:
Right that's why I only put "verify permissions"
we don't know if the current user can actually those permissions for themselves, that's why i kept it generic